SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -130,6 +130,7 @@ class USpatialReceiver : public UObject
 	void OnEntityQueryResponse(Worker_EntityQueryResponseOp& Op);
 
 	void CleanupDeletedEntity(Worker_EntityId EntityId);
+	void ClearRPCsOnEntityCreation(const Worker_EntityId& EntityId);
```

Comment:
Just so we can match the function above and below, you can take the entity Id by non-const value.